### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#171] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-30 171341](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/376791cc-da85-4f24-b977-15a077d1ce76)

**Comment by CiottoloMaggico** [comment, marked as answer]
# Soluzione analoga a quella del prof
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/3a4b5c63-856f-4362-9439-b5bd32dd2458)
